modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/DeployService.java

```diff
@@ -360,6 +369,36 @@ public void purgeService(Deployment deployment, DeployTask deployTask) {
         }
     }
 
+    private void validateDeploymentWithPolicies(Deployment deployment, DeployTask deployTask) {
+
+        if (Objects.isNull(deployTask.getDeployRequest())) {
+            return;
+        }
+        String userId = deployTask.getDeployRequest().getUserId();
+        Csp csp = deployTask.getDeployRequest().getCsp();
+        if (StringUtils.isEmpty(userId) || Objects.isNull(csp)) {
+            log.info("Get user id:{} csp:{} from deploy task.", userId, csp);
```

Comment:
Do we really need this check? Because these two fields will never be empty. 

If we need this, then the log line can be -> UserId is {} Csp is {}. Policy validation stop skipped.

Reply:
I removed this check.